### DISCUSSION TITLE
Help needed with Grafana (Docker, QNap)

### DISCUSSION THREAD

**Discussion by Maximus48p** [NginxProxyManager/nginx-proxy-manager#2120] [Q&A]
Hi,

Been trying to get this to work for hours but no luck so far.
Other applications works flawlessly except Grafana.

Trying to change the settings according these pages:
https://grafana.com/tutorials/run-grafana-behind-a-proxy/#configure-nginx
and
https://github.com/grafana/grafana/issues/36929

I also changed the config file for Grafana according this page:
https://grafana.com/tutorials/run-grafana-behind-a-proxy/#introduction

Any help is really appreciated.

Thx,
Max

**Comment by LukeSkywalker993** [comment]
same here :-(

**Comment by christofkac** [comment]
Hi,
on my side, Grafana works fine behind NginxProxyManager.
Here the NginxSettings:
![Screenshot](https://user-images.githubusercontent.com/33951830/183267764-7af4ecae-a273-433b-87cb-8b3a8bb42785.jpg)
Use use Grafana in a Docker container, here the only setting is required:
![image](https://user-images.githubusercontent.com/33951830/183267807-9664e692-746f-4d58-86a1-316f0a3e2080.png)

What is wrong on your side?
Bye
Christof

**Comment by Maximus48p** [comment]
Thx for you help, unfortunately not working for me.
I have SSL enabled, but even without no luck.

Have you changed Nginx settings within the advanced tab?
No other changes within the grafana.ini file?

**Reply by christofkac** [reply]
The only other setting I changed is the SSL one:
![image](https://user-images.githubusercontent.com/33951830/194386838-b5323b5f-261c-4559-873d-e2c7a49f6af0.png)
In the advanced tab, everything is unchanged.
Grafana.ini is not touched as I run it in a Docker environment, here everything is controlled via environment variables.

What is not working?
Bye
Christof

**Comment by Maximus48p** [comment]
Updated to version 10 of grafana and no issues anymore.

**Comment by LukeSkywalker993** [comment]
Same here. Updated a few days ago.

Working.

thanks!
